docker-compose.yml

```diff
@@ -17,3 +17,8 @@ services:
       - /tmp/.X11-unix:/tmp/.X11-unix
     runtime: nvidia
     tty: true
+    networks:
+      - proxynet
+networks:
+    proxynet:
+      name: turtlenet
```

Comment:
Why this is used and the relevance to the `host = noetic` in `default.ini` is not obvious.

Reply:
As for noetic as host the entire default.ini file was removed in the next commit so it was a setting that was leftover and should have been set back to localhost in that commit.

Reply:
The reason for setting up the network is to have a specific docker network running for the simulation. This enables different docker containers to be started which may connect to the docker container that is running the simulation.

So if we want to start ISAR in a docker container we would be able to connect it to the simulation by saying it should be on an external network named `turtlenet`

Reply:
OK. Are you sure this works across docker-compose files? That would assume we at least run both on the same machine, right?

Reply:
Yes, this works across docker-compose files. I have only tested on the same machine but I think you are correct in that statement.

Reply:
Some adjustments might have to be made once we actually deploy this if we decide to go for multiple VMs in that environment.

Reply:
I suggest we at least rename it to `turtlenet`, I do not see why we have two names for it.